varinfo/generate_umm_var.py

```diff
@@ -0,0 +1,92 @@
+''' This module generates UMM-Var records for a specified collection, and
```

Comment:
This is the main bit of work that I've done. It's nothing too fancy, just wrapping existing functions and bringing them together in one place.